source/git-util.js

```diff
@@ -181,3 +181,16 @@ exports.verifyRecentGitVersion = async () => {
 
 	verifyRequirementSatisfied('git', installedVersion);
 };
+
+exports.checkIfFileGitIgnored = async pathToFile => {
+	try {
+		const {stdout} = await execa('git', ['check-ignore', pathToFile]);
+		return Boolean(stdout);
+	} catch (error) {
+		if (error.stdout === '' && error.stderr === '') {
```

Comment:
This could use a code comment. It's not immediately clear why this check is here.